modeltranslation/utils.py

```diff
@@ -141,12 +147,11 @@ def auto_populate(mode='all'):
         with auto_populate('required'):
             call_command('loaddata', 'fixture.json')
     """
-    current_population_mode = settings.AUTO_POPULATE
-    settings.AUTO_POPULATE = mode
+    set_auto_populate(mode)
     try:
         yield
     finally:
-        settings.AUTO_POPULATE = current_population_mode
+        set_auto_populate(None)
```

Comment:
It isn't "None", it's supposed to revert to previous value.

Reply:
This will remove the value from thread local which will make fallbacks_enabled() and auto_populate_mode() return the value from settings